comp/core/tagger/component.go

```diff
@@ -38,6 +38,7 @@ type Component interface {
 	Stop() error
 	ReplayTagger() ReplayTagger
 	GetTaggerTelemetryStore() *telemetry.Store
+	LegacyTag(entity string, cardinality types.TagCardinality) ([]string, error)
```

Comment:
Can we please add a comment here explaining why we still have this `LegacyTag` method? 

Basically something similar to [this](https://github.com/DataDog/datadog-agent/blob/8de7847264222e3558f7a06c1572b633b183f5fe/comp/core/tagger/global.go#L48-L51):

Maybe something like:

```
// LegacyTag has the same behaviour as the Tag method, but it receives the entity id as a string and parses it. 
// If possible, avoid using this function, and use the Tag method instead.
// This function exists in order not to break backward compatibility with rtloader and python
// integrations using the tagger
```

Reply:
If I am not mistaken, when we command+click on a tagger component instance from another component, it redirects us to the component interface. That's why I would love to this warning comment visible easily to programmers to avoid missing it.